src/transformers/models/phi/modeling_phi.py

```diff
@@ -506,7 +506,7 @@ def forward(
             value_states = value_states.to(target_dtype)
 
         attn_output = self._flash_attention_forward(
-            query_states, key_states, value_states, attention_mask, q_len, dropout=attn_dropout, softmax_scale=1.0
+            query_states, key_states, value_states, attention_mask, q_len, dropout=attn_dropout, softmax_scale=None
         )
```

Comment:
Ah, was this the reason for the issue regarding fine-tuning? 
Now I am curious how the FA tests were passing before...

Anyway thanks a lot for fixing this!

Reply:
BTW just to be sure, could you please run all flash attention tests(for `phi`) to check if they are passing or not.

```
RUN_SLOW=1 pytest -m flash_attn_test tests/models/phi --verbose
```

Should not be necessary since it's already fixing the fine-tuning issue, but just to be sure.

Reply:
Just ran and everything has passed!

Reply:
Yep, our CIs don't test flash attention, bit of a pity!